dbgetdata.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**Pulls info from the database
+ *
+ * @param $connection
+ *
+ * @return mixed
+ */
+function getData($connection) {
+    $connection->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $connection->prepare("SELECT `pick_order`, `picker`, `title`, `author`, `genre` FROM `books`;");
```

Comment:
I'd recommend putting your query into a variable and passing it into the function as a second parameter so you can reuse on other pages and with other SQL queries!